tests/integration/test_governed_evolution_bounded_stop_policy.py

```
@@ -0,0 +1,193 @@
+            return str(Path(candidate))
+    return None
+
+
+def invoke_runtime(*, run_id: str, requested_stage_stop: str, entry_intent_id: str) -> dict:
+    shell = resolve_powershell()
+    if shell is None:
+        raise unittest.SkipTest("PowerShell executable not available in PATH")
+
+    artifact_root = Path(tempfile.mkdtemp(prefix=f"pytest-bounded-stop-{uuid.uuid4().hex}-"))
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)